projects/js-packages/publicize-components/src/components/admin-page/index.tsx

```diff
@@ -52,7 +52,10 @@ export const SocialAdminPage = () => {
 		};
 	}, [] );
 
-	const { social, jetpack } = getSocialScriptData().plugin_info;
+	const {
+		plugin_info: { social, jetpack },
+		has_connected_jetpack_plugin: hasConnectedJetpackPlugin,
```

Comment:
I think we don't need this prop here. We can re-use the existing data to compute

<img width="492" alt="image" src="https://github.com/user-attachments/assets/753766f2-79bb-43c8-9711-24a8b692dd40" />

`jetpack` is already destructured here from `plugin_info`. `jetpack.version` is `null` if the plugin is not active.

```jsx
const showFooterAdminLinks = jetpack.version && JetpackScriptData.connection?.connectionStatus?.hasConnectedOwner;
```

Reply:
Makes sense! The `publicize-components` package depends on the connection JS package, so we can use `JetpackScriptData.connection` directly. Done in https://github.com/Automattic/jetpack/pull/42000/commits/015aa9d7142bddfcca1a51919fe85003996f4986 :+1: